src/Umbraco.Web.UI.Client/src/assets/lang/en.ts

```diff
@@ -438,10 +438,10 @@ export default {
 		compositionDescription:
 			"Defines a re-usable set of properties that can be included in the definition of\n      multiple other Document Types. For example, a set of 'Common Page Settings'.\n    ",
 		folder: 'Folder',
-		folderDescription:
-			'Used to organise the Document Types, Compositions and Element Types created in this\n      Document Type tree.\n    ',
+		folderDescription: 'Used to organize items and other folders. Keep items structured and easy to access.',
```

Comment:
Maybe I have it the wrong way around, but if this is "UK English, then should be "organise" here and "organize" in the US override.

Reply:
You're absolutely right! I missed that. Glaring obvious now you've pointed it out. 🤦😂

I'll get it corrected.